malib/settings.py

```diff
@@ -91,6 +92,8 @@
         # rollout in parallel.
         "num_episodes": 1000,  #
         "episode_seg": 100,
+        "test_num_episodes": 0,
+        "test_episode_seg": 0,
```

Comment:
For evaluation?

Reply:
Yes, this is also used for deterministic evaluation.
Please take a look at:
https://github.com/apexrl/malib/blob/196de6592fd82ea889cb871d4663d9e4d5028dde/malib/manager/rollout_worker_manager.py#L81